### PR TITLE
*: Add the field of history information in job

### PR DESCRIPTION
**Pull request by zimulala**
Handle #1936.
This is for compatibility with previous version. So it is the first step, the second is remove the assignment to the job's arguments.

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @GregoryIan @coocood @tiancaiamao @shenli

File: model/ddl.go [hunk 1]
```diff
@@ -73,6 +73,27 @@ func (action ActionType) String() string {
...
+
+// AddDBInfo adds schema version and schema information that are used for binlog.
+// dbInfo is added in the following operations: create database, drop database.
+func (h *HistoryInfo) AddDBInfo(schemaVer int64, dbInfo *DBInfo) {
+	h.SchemaVersion = schemaVer
+	h.DBInfo = dbInfo
+}
+
+// AddTableInfo adds schema version and table information that are used for binlog.
+// tblInfo is added except for the following operations: create database, drop database.
```
Comment (coocood) [inline model/ddl.go]: create table, drop table.

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
@GregoryIan PTAL

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**